Forge.Forms/src/Forge.Forms/DynamicExpressions/Resource.cs

```diff
@@ -37,6 +41,20 @@ public abstract class Resource : IEquatable<Resource>, IValueProvider
                 ["IsEqual"] = parameter => new IsEqualConverter(parameter)
             };
 
+        public static readonly Dictionary<string, IMultiValueConverter> MultiValueConverters =
+            new Dictionary<string, IMultiValueConverter>(StringComparer.OrdinalIgnoreCase)
+            {
+                ["Append"] = new AppendMultiConverter(),
```

Comment:
I wonder what the difference between this and [dynamic expressions](https://wpf-forge.github.io/Forge.Forms/guides/dynamic-resources#dynamic-expressions) is?

If there is no scenario where one does more than the other, then we may not need this as it may cause confusion?

Reply:
I just created that as an example when I was testing the MultiBindings. I can take it out since it doesn't really serve a purpose.